.gitignore

```diff
@@ -2,7 +2,7 @@
 *.egg-info/
 *.swp
 .eggs/
-c7n/version.py
+# c7n/version.py
```

Comment:
setuptools_scm integration goes away with poetry, we do however gain the ability to automatically rev revision with poetry version command

Reply:
one note on that topic, we'll still need to add the version to version.py manually, although that should be a trivial add to a separate release management tool.